### PR TITLE
Add main menu and stub out options menu

### PR DESCRIPTION
**Pull request by sharkeyjames**
Feast yer eyes on these beauties!

Main menu
image

Options menu [stub]
image

### CODE REVIEW

File: title-screen/TitleScreen.tscn [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+[ext_resource path="res://title-screen/buttons/OptionsButton.tscn" type="PackedScene" id=1]
+[ext_resource path="res://title-screen/flappy-race-logo.png" type="Texture" id=2]
+[ext_resource path="res://title-screen/buttons/NewGameButton.tscn" type="PackedScene" id=3]
+[ext_resource path="res://fonts/DelaGothicOne-24.tres" type="DynamicFont" id=4]
+[ext_resource path="res://icon.png" type="Texture" id=5]
+[ext_resource path="res://title-screen/fade_in/FadeIn.tscn" type="PackedScene" id=6]
+[ext_resource path="res://title-screen/title_screen.gd" type="Script" id=7]
+
+[node name="TitleScreen" type="Control"]
```
Comment (Jibbajabbafic) [inline title-screen/TitleScreen.tscn]: While testing different window sized the content didn't always stay centred but I think you could fix that by making the root node a MarginContainer

File: title-screen/title_screen.gd [hunk 2]
```diff
@@ -0,0 +1,16 @@
+extends Control
```
Comment (Jibbajabbafic) [inline title-screen/title_screen.gd]: Could you move the files into different directories? They should roughly follow this structure:

- .gd -> scripts/
- .png -> sprites/
- .tscn -> scenes/

Feel free to make new directories under those to organise it better, e.g. a ui subdirectory

**Review by Jibbajabbafic — CHANGES_REQUESTED** [review]
Looks really nice for a first version. Love the menu changing animation!
Just had a few things you could change.

**Comment by wildjames** [comment]
This looks fantastic! Nicely done Sharkey

**Review by Jibbajabbafic — APPROVED** [review]

**Pull request merged**